tests/e2e/mgmt_wrkld_domain_isolation.go

```diff
@@ -841,4 +875,316 @@ var _ bool = ginkgo.Describe("[domain-isolation] Management-Workload-Domain-Isol
 			metav1.ListOptions{FieldSelector: fmt.Sprintf("involvedObject.name=%s", pvclaim.Name)}, expectedErrMsg)
 		gomega.Expect(errorOccurred).To(gomega.BeTrue())
 	})
+
+	/*
+		Testcase-3 from Negative scenario
+		Add tests for all supported operations on PVC
+		Ex: Snapshot create/delete, restore snapshot, expand, migrate, attach/detach with zone marked for delete.
+		Use both zonal policy and shared datastore policy
+	*/
+
+	ginkgo.It("Run volume expansion, create and restore operations", func() {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		// statefulset replica count
+		replicas = 3
+
+		// reading zonal storage policy of zone-1 mgmt domain
```

Comment:
This comment is not matching with the fetch export variables, i can see you are reading zone-2 as well